main.go

```diff
@@ -84,7 +84,7 @@ func main() {
 	pflag.BoolVar(&enableLeaderElection, "enable-leader-election", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
-	pflag.StringVar(&collectorImage, "collector-image", fmt.Sprintf("otel/opentelemetry-collector:%s", v.OpenTelemetryCollector), "The default OpenTelemetry collector image. This image is used when no image is specified in the CustomResource.")
+	pflag.StringVar(&collectorImage, "collector-image", fmt.Sprintf("ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector:%s", v.OpenTelemetryCollector), "The default OpenTelemetry collector image. This image is used when no image is specified in the CustomResource.")
```

Comment:
is it available?

i tried (podman/docker):
```bash
run --rm -it otel/opentelemetry-collector # worked

run --rm -it ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector       
Trying to pull ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector:latest...
...
reading manifest latest in ghcr.io/open-telemetry/opentelemetry-collector-releases/opentelemetry-collector: manifest unknown
```

Reply:
https://github.com/open-telemetry/opentelemetry-collector-releases/pkgs/container/opentelemetry-collector-releases%2Fopentelemetry-collector

yes it is, but only since 0.44.0

Reply:
This PR should go in after https://github.com/open-telemetry/opentelemetry-operator/pull/719